plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -803,6 +803,25 @@ private void updateView(ConnectorSession session, SchemaTableName viewName, Conn
         }
     }
 
+    private void updateMaterializedView(ConnectorSession session, SchemaTableName viewName, ConnectorMaterializedViewDefinition newDefinition)
```

Comment:
Can we move this method after `updateMaterializedViewColumnComment` method definition ?